polyform.md

```diff
@@ -14,7 +14,9 @@ The licensor grants you a copyright license for the software, to do everything y
 
 ## DL: Distribution License
 
-The licensor grants you an additional copyright license to distribute copies of the software, and if these terms grant you an additional license to make changes and new works, to distribute your changes and new works, as well.  You must ensure that anyone who gets a copy of any part of the software from you also gets the text of these terms or the URL for its text above.
+The licensor grants you an additional copyright license to distribute copies of the software.  If these terms grant you an additional license to make changes and new works, your license to distribute covers distributing your changes and new works, as well.  However, if these terms allow you to make changes and new works for internal business operations, your license to distribute does not cover distributing those changes and new works.
+
+You must ensure that anyone who gets a copy of any part of the software from you also gets the text of these terms or the URL for its text above.
```

Comment:
Don't need the word `above`?

Reply:
I don't like using "above", but we want to be clear that the URL folks can use is the URL at the top of the license.

Perhaps we put "Official URL:" or "Permalink:" before the URL at the top, and then allow using the "official URL" in this section?

Reply:
SGTM